configuration/wording/WordingFr.tsx

```diff
@@ -77,8 +77,19 @@ export class WordingFr implements Wording {
   readonly AFFICHEZ_RESULTATS_EN_PLAN: string = 'Affichez les résultats sur une carte'
   readonly BESOINS_D_ACCESSIBILITE: string = 'Besoins d’accessibilité '
   readonly MODIFIER_VOTRE_BESOIN_D_ACCESSIBILITE: string = 'Modifier votre besoin d’accessibilité'
-  readonly LIEUX_CORRESPONDENT_A_VOTRE_RECHERCHE: (nombre: number) => string = (nombre: number): string => nombre > 1 ? `${nombre} lieux correspondent à votre recherche` : `${nombre} lieu correspond à votre recherche`
-  readonly AUCUN_LIEU_NE_CORRESPOND_A_VOTRE_RECHERCHE: string = 'Aucun lieu ne correspond à votre recherche.'
+  readonly LIEUX_CORRESPONDENT_A_VOTRE_RECHERCHE: (nombre: number, rayonDeRecherche: boolean) => string =
+    (nombre: number, rayonDeRecherche: boolean): string => {
```

Comment:
```suggestion
    (nombre: number, hasRayonDeRecherche: boolean): string => {
```
Pareil dans tout le code.